src/main/resources/signup.html

```
@@ -0,0 +1,67 @@
+<<!doctype html>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)